src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.AspNet.Identity;
+
+namespace Microsoft.Framework.Logging
```

Comment:
The guidelines are i believe that the namespace for the extension methods be the same the class they are on ? In this case we don't want to the `.Log(IdentityResult)` to show up on the ILogger in places where Identity is not being used. I have conflicting views here but I still prefer keeping it in `Microsoft.Framework.Logging`